src/fluree/db/ledger/mutable.clj

```diff
@@ -6,7 +6,9 @@
             [fluree.db.storage.core :as storage]
             [fluree.db.constants :as const]
             [fluree.db.util.log :as log]
-            [fluree.db.ledger.reindex :as reindex])
+            [fluree.db.ledger.reindex :as reindex]
+            [fluree.db.util.core :as util]
+            [fluree.db.flake :as flake])
   (:import (fluree.db.flake Flake)))
```

Comment:
I know this isn't something you added here, but we no longer need the `Flake` type hint, we've got `s` `p` `o` `t` etc helpers in the `flake` namespace that do the hinting for us.